ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -470,7 +433,9 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
     }
 
     if !userKeyboards.contains(where: { $0.id == kb.id }) {
-      let keyboardDir = Storage.active.keyboardDir(forID: kb.id)
+      // TODO:  should make sure that there are no resources in the package,
+      //        rather than just 'no matching keyboards'.
```

Comment:
This is best left for later work; this PR is complex enough as it is, and fixing this would add even more complexity.  On approval, I'll make an issue for it.

It's probably low priority - this concern only affects multi-keyboard packages when a user fully uninstalls a single contained keyboard.  It's not exactly the most common case, as it should only occur for manually-shared KMPs.  (Our language searches don't return multi-keyboard packages, right?)

Reply:
In a language search for "ancient hebrew"
https://keyman.com/keyboards/languages/hbo

The results include the Galaxie Greek/Hebrew (mnemonic) multi-keyboard package
https://keyman.com/keyboards/galaxie_greek_hebrew_mnemonic